File: csharp/ex_078_sum-of-multiples/SumOfMultiples.cs
using System.Linq;
using System.Collections.Generic;

public static class SumOfMultiples
{
    public static int Sum(IEnumerable<int> multiples, int max)
    {
        HashSet<int> bag = new HashSet<int>();
        foreach(int n in multiples)
            if(n > 0)
                for(int i = n; i < max; i += n)
                    bag.Add(i);
        return bag.Sum();
    }
}

File: javascript/ex_119_sum-of-multiples/sum-of-multiples.js
export const sum = (base, lvl) => {
  const bag = new Set();
  base.forEach(n => {
    if(n > 0) for (let i = n; i < lvl; i += n)
        bag.add(i);
  });
  return [...bag].reduce((acc, e) => acc + e, 0);
};